.github/workflows/examples-java-push.yml

```diff
@@ -55,3 +55,24 @@ jobs:
           name: examples-java-reports
           path: '**/build/reports/*'
           retention-days: 5
+  deploy:
+    environment: sandbox
+    runs-on: ubuntu-18.04
+    needs: test
+    if: github.ref == 'refs/heads/main'
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
```

Comment:
We don't need the full history, only a shallow clone here:
```suggestion
```

(This is needed elsewhere for SonarQube)